modules/fbs/includes/fbs.reservation.inc

```diff
@@ -326,10 +326,24 @@ function fbs_reservation_branch_name($branch_id) {
  *   The default options.
  */
 function fbs_reservation_default_options($account) {
+  // Load profile2 profile.
+  $profile = ding_user_provider_profile($account);
+
+  // Use a metadata wrapper to access the data.
+  $wrapper = entity_metadata_wrapper('profile2', $profile);
+
+  // Get values.
+  $interest_period = $wrapper->field_fbs_interest_period->value();
+
+  if (empty($interest_period)) {
+    $interest_period = fbs_reservation_default_interest_period();
+  }
+
```

Comment:
The lines above that looks in the profile for the periode, looks like do the same "fbs_reservation_default_interest_period($account) would do before returning the default periode.

So why do this above when the function do the same?

Reply:
Good point...